src/core/linalg/src/dense/4C_linalg_fixedsizematrix.hpp

```diff
@@ -1915,6 +1915,11 @@ namespace Core::LinAlg
     double* values(Core::LinAlg::SerialDenseVector& vector);
   }  // namespace Internal
 
+  enum class Initialization
+  {
+    set_zero,
+    leave_uninitialized
```

Comment:
Why not shorter with the same meaning and expressiveness?
```suggestion
    zero,
    none
```
or 
```suggestion
    zero,
    uninitialized
```
or
```suggestion
    true,
    false
```
or
```suggestion
    yes,
    no
```

Admittedly, the last two "bool"-style variants would not allow to be extended to other initialization cases. Are there plans or ideas for extension?

Reply:
I like `zero` and `uninitialized`.